python/lammps/data.py

```diff
@@ -11,82 +11,83 @@
 #   See the README file in the top-level LAMMPS directory.
 # -------------------------------------------------------------------------
 
-################################################################################
-# LAMMPS data structures
-# Written by Richard Berger <richard.berger@temple.edu>
-################################################################################
+"""
+Data structures for LAMMPS Python module
+Written by Richard Berger <richard.berger@temple.edu>
```

Comment:
If you're touching these lines, might as well update this to `richard.berger@outlook.com`